Model/Page.php

```diff
@@ -134,6 +134,11 @@
     protected $target;
 
     /**
+     * @var int
+     */
+    protected $targetRedirectType = 302;
```

Comment:
Hmmm nevermind, the doctrine docs seem to suggest otherwise http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/inheritance-mapping.html